src/explanations/intro-to-defi.yaml

```
@@ -1,6 +1,118 @@
+- title: What is DeFi?
+  shortTitle: What is DeFi
+  key: what-is-defi
+  details: |
+   Decentralized finance, or "DeFi" for short, is the concept of a financial institution built on blockchain technology that is not controlled by a single entity. Instead, it is governed by a set of rules that can be modified by users. This allows for a more democratic approach to 
+   banking, as there is no central point of power. A DeFi platform is different from traditional financial institutions because it doesn't have people working behind the scenes to run things - instead, it relies on code-based logic. These blocks of code are called smart contracts, 
+   and they're deployed on the blockchain. One key feature of smart contracts is that the data they write on the chain is immutable, which means it can't be changed once the transactions are settled. 
+   To the DeFi protocol, every blockchain address is an entity. As long as the entity satisfies the conditions that the smart contract programmer puts forth, it's allowed to perform the desired action.
+
+
 - title: Why do we need DeFi?
   shortTitle: Why DeFi
   key: why-do-we-need-defi
   details: |
-    - Why do we need DeFi ?
-    This needs to be explained in a detailed way
+   Banks have been around for a very long time, but the rate of innovation in the banking sector is quite slow. In the past decade, 
+   many things have changed, including:
+
+   1) Our economy is now a global economy, instead of being limited to just one country.
+   2) People prefer online transactions that are instant, instead of in-person banking. 
+   3) Need for more flexibility in how funds can be collected and distributed.
+
+   DeFi (decentralized finance) aims to provide an alternative to traditional banking that comes up with solutions for existing flaws in the banking system. DeFi is based on blockchain technology and offers the following solutions to problems faced in TradFi.
+   
+   **Innovation:**
```

```suggestion
   #### Innovation:
```
Please change the formatting at other places also. ** just makes it bold. We also want to move the text to next line. So, for that we can use ####